server/logger.js

```diff
@@ -12,26 +12,26 @@ const logger = {
 
   // Called whenever there's an error on the server we want to print
   error: err => {
-    console.log(chalk.red(err));
+    console.error(chalk.red(err));
   },
 
   // Called when express.js app starts on given port w/o errors
   appStarted: (port, tunnelStarted) => {
-    console.log('Server started ' + chalk.green('✓'));
+    console.log(`Server started ${chalk.green('✓')}`);
 
     // If the tunnel started, log that and the URL it's available at
     if (tunnelStarted) {
-      console.log('Tunnel initialised ' + chalk.green('✓'));
+      console.log(`Tunnel initialised ${chalk.green('✓')}`);
     }
 
     console.log(
-      chalk.bold('\nAccess URLs:') +
-      divider +
-      '\nLocalhost: ' + chalk.magenta('http://localhost:' + port) +
-      '\n      LAN: ' + chalk.magenta('http://' + ip.address() + ':' + port) +
-      (tunnelStarted ? '\n    Proxy: ' + chalk.magenta(tunnelStarted) : '') +
-      divider,
-      chalk.blue('\nPress ' + chalk.italic('CTRL-C') + ' to stop\n')
+      chalk.bold(`
```

Comment:
Yeah, good catch! I haven't tried it, but that sounds like a likely error–does that happen @glennreyes?